install.sh

```diff
@@ -318,7 +318,7 @@ install_kubectl() {
     err_msg=$(kubectl version --client=true 2>&1 1>/dev/null)
     if [ "$err_msg" == "" ]; then
         v=$(kubectl version --client=true | sed 's/.*GitVersion:\"v\([0-9.]*\).*/\1/g')
-        target_version=$(echo "${kubectl_version}" | sed s/v//g)
+        target_version=${kind_version//v/g}
```

Comment:
```
#!/usr/bin/env bash

kubectl_version="v0.11.1"
target_version=$(echo "${kubectl_version}" | sed s/v//g)
echo ${target_version}
target_version=${kubectl_version//v/g}
echo ${target_version}
```
I have write a test script , but it failed with:./test.sh: 6: ./test.sh: Bad substitution

Reply:
#!/usr/bin/env bash

```
kubectl_version="v0.11.1"
target_version=$(echo "${kubectl_version}" | sed s/v//g)
echo ${target_version}
target_version=${kubectl_version//v}
echo ${target_version}
```
Returns ->
```
0.11.1
0.11.1
```